src/client/transaction.gen.go

```diff
@@ -471,6 +417,20 @@ func (c *unsupportedPpsBuilderClient) ListTask(_ context.Context, _ *taskapi.Lis
 	return nil, unsupportedError("ListTask")
 }
 
+type unsupportedDebugBuilderClient struct{}
```

Comment:
this file should also be reverted right?

Reply:
Regenerated, I think, rather than reverted. I ran `make proto` again, and am adding a commit with its updates.